### PR TITLE
Release 3.2.0 with ENABLE_LLVM=on.

### PR DESCRIPTION
**Pull request by tcmoore3**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Review by b-butler — APPROVED** [review]

**Comment by joaander** [comment]
@conda-forge-admin, please restart ci

**Comment by tcmoore3** [comment]
The upstream issue is addressed in glotzerlab/hoomd-blue#1321

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% set processor = "cpu" %}
 {% set name = "hoomd" %}
 {% set version = "3.1.0" %}
-{% set build = 0 %}
+{% set build = 1 %}
```
Comment (joaander) [inline recipe/meta.yaml:4]:
```suggestion
{% set build = 0 %}
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -12,9 +12,11 @@ package:
...
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://github.com/glotzerlab/hoomd-blue/releases/download/v{{ version }}/{{ name }}-v{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+#  fn: {{ name }}-{{ version }}.tar.gz
+#  url: https://github.com/glotzerlab/hoomd-blue/releases/download/v{{ version }}/{{ name }}-v{{ version }}.tar.gz
+#  sha256: {{ sha256 }}
+  git_url: https://github.com/glotzerlab/hoomd-blue
+  git_rev: fix/find-clang
```
Comment (joaander) [inline recipe/meta.yaml:19]:
```suggestion
 fn: {{ name }}-{{ version }}.tar.gz
 url: https://github.com/glotzerlab/hoomd-blue/releases/download/v{{ version }}/{{ name }}-v{{ version }}.tar.gz
 sha256: {{ sha256 }}
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {% set processor = "cpu" %}
 {% set name = "hoomd" %}
 {% set version = "3.1.0" %}
```
Comment (joaander) [inline recipe/meta.yaml:3]:
```suggestion
{% set version = "3.2.0" %}
```

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
-{% set build = 0 %}
+{% set build = 1 %}
 {% set sha256 = "4a83d3d6098f07cb0deaacd2953998abeb111aad98fe5a0eb9403001d58d704c" %}
```
Comment (joaander) [inline recipe/meta.yaml:5]: Set the correct sha256 after the v3.2.0 release tarball is uploaded.

**Review by joaander — CHANGES_REQUESTED** [review]
Changes needed after the v3.2.0 release.

File: recipe/meta.yaml [hunk 5]
```diff
@@ -12,9 +12,9 @@ package:
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://github.com/glotzerlab/hoomd-blue/releases/download/v{{ version }}/{{ name }}-v{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+ fn: {{ name }}-{{ version }}.tar.gz
+ url: https://github.com/glotzerlab/hoomd-blue/releases/download/v{{ version }}/{{ name }}-v{{ version }}.tar.gz
+ sha256: {{ sha256 }}
```
Comment (joaander) [inline recipe/meta.yaml:17]:
```suggestion
  fn: {{ name }}-{{ version }}.tar.gz
  url: https://github.com/glotzerlab/hoomd-blue/releases/download/v{{ version }}/{{ name }}-v{{ version }}.tar.gz
  sha256: {{ sha256 }}
```

**Review by joaander — CHANGES_REQUESTED** [review]
No need to remove spaces.

**Comment by joaander** [comment]
@conda-forge-admin, please rerender

**Review by joaander — APPROVED** [review]
Thanks!

**Pull request merged**